modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -18,13 +18,11 @@
 			<param index="2" name="b8" type="int" />
 			<param index="3" name="a8" type="int" default="255" />
 			<description>
-				Returns a color constructed from integer red, green, blue, and alpha channels. Each channel should have 8 bits of information ranging from 0 to 255.
-				[code]r8[/code] red channel
-				[code]g8[/code] green channel
-				[code]b8[/code] blue channel
-				[code]a8[/code] alpha channel
+				Returns a [Color] constructed from red ([param r8]), green ([param g8]), blue ([param b8]), and optionally alpha ([param a8]) integer channels, each divided by [code]255.0[/code] for their final value.
 				[codeblock]
-				red = Color8(255, 0, 0)
+				var red = Color8(255, 0, 0)            # Same as Color(1, 0, 0)
+				var dark_blue = Color8(0, 0, 51)       # Same as Color(0, 0, 0.2).
+				var my_color = Color8(306, 255, 0, 63) # Same as Color(1.2, 1, 0, 0.8).
```

Comment:
```suggestion
				var my_color = Color8(306, 255, 0, 204) # Same as Color(1.2, 1, 0, 0.8).
```

Reply:
Oops my intention was to use the lower integer but I didn't update the equivalent.